fullstack-cert/react-projects/usememo-currency-converter/userstories.md

```diff
@@ -0,0 +1,13 @@
+**User Stories:**  
+
+1. You should create and export a `CurrencyConverter` component.  
```

Comment:
I think at this point it is fine to provide the component and any necessary imports for them. That is what the other labs are doing by this point in the curriculum.

 ```suggestion
```